docs/sources/trackers/google-tag-manager/tag-types/index.md

```diff
@@ -0,0 +1,302 @@
+| `advertiserId` | No        | The advertiser ID                                                      | 201     |
+| `campaignId`   | No        | The campaign ID                                                        | 12      |
+| `cost`         | No        | The cost of the ad                                                     | 5.5     |
+| `costModel`    | No        | The cost model for the campaign. Must be one of `cpc`, `cpm`, or `cpa` | cpc     |
+
+
+
+The ad tracking tag includes three event types, each with its own set of additional parameters:
+
+#### Impression Event
+
+| Name           | Required? | Description                                                      | Example                 |
+| -------------- | --------- | ---------------------------------------------------------------- | ----------------------- |
+| `impressionId` | No        | Identifier for the particular impression instance                | 67965967893             |
+| `targetUrl`    | No        | The destination URL                                              | https://www.example.com |
+| `bannerId`     | No        | Adserver identifier for the ad banner (creative) being displayed | 23                      |
+| `zoneId`       | No        | Adserver identifier for the zone where the ad banner is located  | 7                       |
+
+#### Click Event
+
+| Name        | Required? | Description                                                      | Example                 |
+| ----------- | --------- | ---------------------------------------------------------------- | ----------------------- |
+| `targetUrl` | Yes       | The destination URL                                              | https://www.example.com |
+| `clickId`   | No        | Identifier for the particular click instance                     | 12243253                |
+| `bannerId`  | No        | Adserver identifier for the ad banner (creative) being displayed | 23                      |
+| `zoneId`    | No        | Adserver identifier for the zone where the ad banner is located  | 7                       |
+
+#### Conversion Event
+
+| Name           | Required? | Description                                       | Example   |
+| -------------- | --------- | ------------------------------------------------- | --------- |
+| `conversionId` | No        | Identifier for the particular conversion instance | 743560297 |
+| `category`     | No        | Conversion category                               | ecommerce |
+| `action`       | No        | The type of user interaction                      | purchase  |
+| `property`     | No        | Describes the object of the conversion            | shoes     |
+| `initialValue` | No        | How much the conversion is initially worth        | 99        |
+
+</details>
+
+
+### Button Click Tracking
+
+This tag enables the automatic tracking of clicks on buttons, covering both `<button>` and `<input type="button">` elements.
```

Comment:
Needs a note that this enables listening for clicks by the plugin, it doesn't track an actual button click event, which GTM users might expect (since GTM has built in triggers for detecting such clicks; a common misconfiguration with the link/form commands).